corehq/motech/dhis2/templates/dhis2/dataset_map_json.html

```diff
@@ -0,0 +1,52 @@
+{% extends "hqwebapp/base_section.html" %}
+{% load i18n %}
+{% load hq_shared_tags %}
+
+{# Required to define ACE #}
+{% load compress %}
+{% block js %}{{ block.super }}
+  {% compress js %}
+    <script src="{% static 'ace-builds/src-min-noconflict/ace.js' %}"></script>
+    <script src="{% static 'ace-builds/src-min-noconflict/mode-json.js' %}"></script>
+    <script src="{% static 'ace-builds/src-min-noconflict/mode-xml.js' %}"></script>
```

Comment:
I don't, but the console shows an error when **base_ace.js** tries to [import it here](https://github.com/dimagi/commcare-hq/blob/1637ae2349ec1d7069d0ff56def39990e724970e/corehq/apps/hqwebapp/static/hqwebapp/js/base_ace.js#L6). So I included it to keep it quiet.

> She's gone tomorrow, boy
> All that she wants
> Is another import

(Or [something](https://www.youtube.com/watch?v=UD2iCuDZTBU) like that.)

Reply:
gotcha.

Reply:
I think something is amiss here, because RequireJS pages shouldn't be using script tags. It looks like this file no longer exists in master - is this code gone now? If not, I'd be happy to help take a look at these dependencies.

Reply:
Thanks for reviewing @orangejenny . Yeah, this PR was merged into `nh/dhis2/sqldataset_maps`, which is in PR https://github.com/dimagi/commcare-hq/pull/28912

This doesn't seem to be the only place this happens:
```
(master) $ git grep 'ace-builds/src-min-noconflict/mode-json.js' | wc -l
5
```

Reply:
The script tags in formplayer and userreports are okay, because those areas don't use requirejs, but the ones in motech shouldn't be necessary. I think something isn't right with the `base_ace.js`. I experimented with removing the script tags and tweaking the ace config (thinking [this](https://stackoverflow.com/questions/31211331/how-to-load-ace-editor-theme-from-cdn-with-requirejs) might be the issue) but I haven't been able to get something that works on staging. It doesn't throw errors, and the syntax highlighting works, but the json syntax checking does not. It's annoying that this isn't working without including the script tags, but given that it isn't causing any user-facing problems, I'm going to leave it alone for now.

Reply:
> I think something isn't right with the base_ace.js. 

Interesting.

> It's annoying that this isn't working without including the script tags, but given that it isn't causing any user-facing problems, I'm going to leave it alone for now.

OK.

Thank you very much for digging into this.